tests/network.rs

```diff
@@ -35,20 +32,29 @@ fn create_sync_req(start_height: u64) -> hotstuff_rs::messages::Message {
     hotstuff_rs::messages::Message::SyncMessage(test_message)
 }
 
-//TODO jonas update tests
 // - Network: Node1, Node2
 // - Node1: keep broadcasting Mempool topic message
 // - Node2: set Node1 as bootnode, listens to subscribed topics
 #[tokio::test]
 async fn test_broadcast() {
-    let (address_1, node_1, _) = node(30001, vec![], None, vec![]).await;
-    let (_address_2, _node_2, receiver_gate) = node(
+    let keypair_1 = ed25519::Keypair::generate();
```

Comment:
Is there a reason why these can't be done inside node() and return it from the function? 
Personally I think duplicated code doesn't look good

Reply:
I have generated the keypair outside of node() to provide flexibility. 

Previously, if node1 was created first, its boot_nodes has to be blank as address_2 and address_3 has yet to come into scope. By generating the keypair outside of node() , we can create linked networks where node1 can set other nodes as bootnodes instead of via random walk() discovery.